de.dlr.sc.virsat.model.extension.visualisation/src/de/dlr/sc/virsat/model/extension/visualisation/treemanager/networking/ZMQSocket.java

```diff
@@ -51,7 +51,6 @@ public void open(String endpoint, int type, boolean bindSockets) {
 			}
 		} catch (Exception e) {
 			activator.getLog().log(new Status(Status.ERROR, Activator.getPluginId(), "Error during port binding.", e));
-			System.exit(-1);

```

Comment:
Good one 😅

Reply:
I think it totally makes sense... We should add this to all our catch statements 🤣